### ISSUE TITLE
Validate metadata columns against config

### ISSUE THREAD

**Issue by fengelniederhammer** [GenSpectrum/LAPIS-SILO#103]
Make sure that we only read columns that are configured in the config. How do we handle columns that are present in the metadata but not in the config? Do we ignore them?

**Comment by chaoran-chen** [comment]
Yes, ignore them and maybe print a message that additional columns were found.
